adventureGame/AdventureGame/test/test_UserInput.py

```diff
@@ -0,0 +1,33 @@
+"""
+@author: Alexander Downing
+"""
+
+import pytest as assertion
+# Allows us to create obj for testing.
+from src import Main as main
+
+def walking_works():

```

Comment:
As part of the pytest framework, all tests must have test_ prefixed to them. e.g. test_walking_works() so that the unit test framework can pick it up